qiskit/test/mock/utils/configurable_backend_v2.py

```diff
@@ -0,0 +1,183 @@
+            qubit_coordinates: Optional specification of grid for displaying gate maps.
+            qubit_t1: Longitudinal coherence times.
+            qubit_t2: Transverse coherence times.
+            qubit_frequency: Frequency of qubits.
+            qubit_readout_error: Readout error of qubits.
+            single_qubit_gates: List of single qubit gates for backend properties.
+            dt: Discretization of the input time sequences.
+            std: Standard deviation of the generated distributions.
+            seed: Random seed.
+        """
+        # harcoded values taken from FakeMumbaiV2 and ConfigurableBackend
+
+        np.random.seed(seed)
+
+        if std is None:
+            std = 0.01
+
+        if measurable_qubits is None:
+            measurable_qubits = list(range(n_qubits))
+
+        if not isinstance(qubit_t1, list):
+            qubit_t1 = np.random.normal(
+                loc=qubit_t1 or 113.0, scale=std, size=n_qubits
+            ).tolist()
+
+        if not isinstance(qubit_t2, list):
+            qubit_t2 = np.random.normal(
+                loc=qubit_t1 or 150.2, scale=std, size=n_qubits
+            ).tolist()
+
+        if not isinstance(qubit_frequency, list):
+            qubit_frequency = np.random.normal(
+                loc=qubit_frequency or 4.8, scale=std, size=n_qubits
+            ).tolist()
+
+        if not isinstance(qubit_readout_error, list):
+            qubit_readout_error = np.random.normal(
+                loc=qubit_readout_error or 0.04, scale=std, size=n_qubits
+            ).tolist()
+
+        if dt is None:
+            dt = 0.2222222222222222e-9
+
+        if not qubit_coordinates is None:
+            self.qubit_coordinates = qubit_coordinates
+
+        self.backend_name = name
+        self.description = description
+        self.gate_configuration = gate_configuration
+        self.measurable_qubits = measurable_qubits
+        self.qubit_t1 = qubit_t1
+        self.qubit_t2 = qubit_t2
+        self.qubit_frequency = qubit_frequency
+        self.qubit_readout_error = qubit_readout_error
+        self.single_qubit_gates = single_qubit_gates
+        self.std = std
+        # self.n_qubits = n_qubits
+
+        super().__init__(
+            name=self.backend_name,
+            description=self.description,
+            online_date=datetime.datetime.utcnow(),
+            backend_version="0.0.0",
+        )
+
+        self._target = Target(dt=dt)
+
+        # Add gates to target
+        # TODO: dynamic instruction properties
+        for gate, qubit_tuple_list in self.gate_configuration.items():
+            temp_gate_props = {
+                qubit_tuple: InstructionProperties(duration=0.0, error=0)
+                for qubit_tuple in qubit_tuple_list
+            }
+            if gate in parameterized_gates.keys():
+                self._target.add_instruction(
+                    gate(Parameter(parameterized_gates[gate])), temp_gate_props
```

Comment:
What about gates with more than 1 parameter. For example, if I want to use a `U` which has 3 parameters I don't think this would work because I can't specify the 3 different parameter names.

Reply:
In instantiation, pass paramerized gates dictionary with values as a list
```python
parameterized_gates={U3Gate: ["theta", "phi", "lam"], RYGate: ["theta"]},
```
```suggestion
                    gate(*[Parameter(p) for p in parameterized_gates[gate]]),
                    temp_gate_props,
                
```